orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
No, suppose the user defined a function which takes an arg of `CacheType` or a struct with `CacheType` type:
```
type MyStruct struct {
CacheType pilosa.CacheType
}
```
Even if we change `CacheType` from `string` to `int`, the code wouldn't break (if they use the `CacheType("lru")` style, they are on their own). If `CacheType` was defined as a string, obviously their code would break.